treeherder/webapp/api/infra_compare.py

```diff
@@ -0,0 +1,42 @@
+import datetime
+import time
+
+from rest_framework import generics
+from treeherder.model import models
+
+from .infra_serializers import InfraCompareSerializer, InfraCompareQuerySerializers
+
+from rest_framework.response import Response
+from rest_framework.status import HTTP_400_BAD_REQUEST
+
+
+class InfraCompareView(generics.ListAPIView):
+    serializer_class = InfraCompareSerializer
+    queryset = None
+
+    def list(self, request):
+        query_params = InfraCompareQuerySerializers(data=request.query_params)
+        if not query_params.is_valid():
+            return Response(data=query_params.errors, status=HTTP_400_BAD_REQUEST)
+
+        startday = query_params.validated_data['startday']
+        endday = query_params.validated_data['endday']
+        project = query_params.validated_data['project']
+        revision = query_params.validated_data['revision']
+        interval = query_params.validated_data['interval']
+        repository = models.Repository.objects.get(name=project)
+        push = models.Push.objects.filter(repository=repository).first()
+        job = models.Job.objects.filter(push=push)
```

Comment:
is job a single job, or a list of jobs?

Reply:
There are a list of jobs. Is there some fault in it

Reply:
can you make it jobs instead of job?